src/main/java/uk/gov/hmcts/dm/service/Constants.java

```diff
@@ -4,4 +4,12 @@
 
     String IS_ADMIN = "isAdmin";
 
+    Boolean FALSE = false;
+
+    String CASE_REF_REGEX = "^\\d{16}$";
```

Comment:
I am not sure if we have similar 16 digits validation for CaseRef across all applications? If, yes. Then we can leave this number to 16.

Reply:
This is specified as part of the end-point design, case references are 16 digist long:
https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1545344961&__ncforminfo=SrXn1AZ7tEZMTraGYHoZrDgvdLA2l7NIkdJFSu8_8di0rnFhv5-CzAMBFusZ-bWRTKI7PtR_T4FGgtIxRx3hdjH5w188wYVr